DAO.sol

```diff
@@ -615,6 +627,45 @@ contract DAO is DAOInterface, Token, TokenCreation {
         p.open = false;
     }
 
+    function withdraw() noEther onlyTokenholders returns (bool _success) {
+
+        // don't allow withdrawing when token holder is blocked due to a vote
+        // unVoteAll() would be an alternative
+        if (getOrModifySplitBlocked(msg.sender))
+            throw;
+
+        // Move ether
+        uint senderBalance = balances[msg.sender];
+        uint fundsToBeMoved = (senderBalance * actualBalance()) / totalSupply;
+        balances[msg.sender] = 0;
+        msg.sender.send(fundsToBeMoved);
```

Comment:
possible stack depth attack here. You need to wrap in if loop. I have fix but cannot push it to thedao repo.